schemas/ingest-metrics.v1.schema.json

```diff
@@ -40,7 +40,79 @@
             "type": "string"
           }
         },
-        "value": {},
+        "value": {
+          "anyOf": [
+            {
+              "title": "encoded_series_metric_value",
```

Comment:
Should we include the `array` format name here?

I would define each possible format separately in the schema as part of the `anyOf` and not mix fields. Slightly more duplication but a lot cleaner separation. E.g. how I sketched in the epic:

```
{
  "anyOf": [
    {
      "type": "object",
      "properties": {
        "format": { "const": "array" },
        "data": { "type": "array", "items": { "type": "number" } }
      }
    },
    {
      "type": "object",
      "properties": {
        "format": { "const": "zstd" },
        "data": { "type": "string" }
      }
    },
    ... gauges, counters, legacy sets and distributions ...
  ]
}
```

Reply:
Ah I see, I might have missed that point when discussing this with you. Updating this now